documentation/modules/exploit/linux/local/cve_2021_4034_pwnkit_lpe_pkexec.md

```diff
@@ -0,0 +1,319 @@
+## Vulnerable Application
+
+The `check` method uses the exploit without a payload.  The patched version
+will exit without any arguments and give the help menu.  Unpatched versions
+give the error that the shell was not in the shells folder.
+
+Last Vulnerable Ubuntu packages:
+- 20.04: 0.105-26ubuntu1.2
```

Comment:
Pretty sure that `0.105-26ubuntu1.2` is the last vulnerable package on Ubuntu 20.04

```suggestion
- 20.04: 0.105-26ubuntu1.1
```

Reply:
This one's on my poor reading comprehension.  It appears these are the first patched versions, not last vulnerable, updating the docs accordingly.